edgelet/iotedge/src/config/mod.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+pub mod apply;
+
```

Comment:
both `apply` and `import` are commands, so you can remove this whitespace and lump the two together

Reply:
Done.